addon/@ember/test-waiters/waiter-manager.ts

```diff
@@ -3,7 +3,40 @@ import { PendingWaiterState, Waiter, WaiterName } from './types';
 import Ember from 'ember';
 import { registerWaiter } from '@ember/test';
 
-const WAITERS: Map<WaiterName, Waiter> = new Map<WaiterName, Waiter>();
+type Indexable = Record<any, unknown>;
+
+// this ensures that if @ember/test-waiters exists in multiple places in the
+// build output we will still use a single map of waiters (there really should
+// only be one of them, or else `settled` will not work at all)
+const WAITERS: Map<WaiterName, Waiter> = (function () {
+  const HAS_SYMBOL = typeof Symbol !== 'undefined';
+
+  let symbolName = 'LIFELINE_QUEUED_POLL_TASKS';
```

Comment:
We should probably rename this symbol.

How about:

```suggestion
  let symbolName = 'TEST_WAITERS';
```

Reply:
lolol copy pasta!!!!!